packages/vue/src/components/organisms/SfFooter/SfFooter.html

```diff
@@ -0,0 +1,10 @@
+<footer>
+    <div class="sf-footer-mobile">
```

Comment:
SfFooter should have 1 slot as in the initial description. I think you must've misunderstood my directions, sorry. The idea was to have SfFooterColumn as component taht displayes either Accordion or just a content with title, so the API would look liek this:
```html
<SfFooter>
  <SfFooterColumn title="Title 1">
    content
  </SfFooterColumn>
  <SfFooterColumn title="Title 1">
    content
  </SfFooterColumn>
</SfFooter>

Reply:
Hey @filrak Yes, I didn't completely understand your directions. Right now it is working with this API

Reply:
Thank you!